doc/source/user_guide/reshaping.rst

```diff
@@ -472,14 +472,15 @@ If ``crosstab`` receives only two Series, it will provide a frequency table.
     pd.crosstab(df['A'], df['B'])
 
 Any input passed containing ``Categorical`` data will have **all** of its
-categories included in the cross-tabulation, even if the actual data does
-not contain any instances of a particular category.
+categories included in the cross-tabulation while setting ``dropna=False``,
+even if the actual data does not contain any instances of a particular category.
 
 .. ipython:: python
 
     foo = pd.Categorical(['a', 'b'], categories=['a', 'b', 'c'])
     bar = pd.Categorical(['d', 'e'], categories=['d', 'e', 'f'])
     pd.crosstab(foo, bar)
+    pd.crosstab(foo, bar, dropna=False)
```

Comment:
can u put this one in a separate ipython block and put the comment before that one